pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/SegmentStatusCheckerTest.java

```diff
@@ -303,7 +303,9 @@ public void missingIdealTest() throws Exception {
     Assert.assertEquals(controllerMetrics.getValueOfTableGauge(tableName,
         ControllerGauge.SEGMENTS_IN_ERROR_STATE), 0);
     Assert.assertEquals(controllerMetrics.getValueOfTableGauge(tableName,
-        ControllerGauge.NUMBER_OF_REPLICAS), 0);
+        ControllerGauge.NUMBER_OF_REPLICAS), 1);
```

Comment:
Need test case for 0 segments in the table. Ideal state exists but number of partitions of the resource are 0.